docs/source/getting_started.rst

```diff
@@ -13,21 +13,46 @@ If you're unfamiliar with the concepts of static and dynamic type checking,
 be sure to read this chapter carefully, as the rest of the documentation
 may not make much sense otherwise.
 
-Installing and running mypy
-***************************
+Installing mypy
+***************
 
-Mypy requires Python 3.9 or later to run.  You can install mypy using pip:
+Mypy requires Python 3.9 or later to run, and can be installed through either pip or uv.
```

Comment:
I'm afraid this is a bit unfair. There is `poetry`, there is `pdm`, there is `pipx` - are we going to enumerate all package managers here? `pip` is the "default" one, so mentioning `pip` is reasonable. Everything else is user's homework IMO: "how to run a tool using your package manager?" I'm not a maintainer here, so please don't treat the comment above as a binding disapproval, let's wait for someone from the core team to comment.